Lib/sre_constants.py

```diff
@@ -21,6 +21,17 @@
 # should this really be here?
 
 class error(Exception):
+    '''Exception raised for invalid regular expressions.
+
+    The error instance has the following additional attributes:
+
+    msg: The unformatted error message
+    pattern: The regular expression pattern
+    pos: The index of pattern where compilation failed
```

Comment:
Leave 'of' alone then.  My comment on 'of' to 'in' was about a 60% change - 40% leave alone  judgment.  But your previous comment reveals that failure can occur during matching (which is the re equivalent of execution).    Source lineno and colno are only relevant when failure *is* during matching.